middleware/envvar/envvar.go

```diff
@@ -0,0 +1,85 @@
+package envvar
+
+import (
+	"encoding/json"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/utils"
+	"os"
+	"strings"
+)
+
+// Config defines the config for middleware.
+type Config struct {
+	// Path specifies the path of handler that exposes environment variables
+	Path string
+	// ExportVars specifies the environment variables that should export
+	ExportVars map[string]string
+	// ExcludeVars specifies the environment variables that should not export
+	ExcludeVars map[string]string
+}
+
+type EnvVar struct {
+	Vars map[string]string `json:"vars"`
+}
+
+func (envVar EnvVar) set(key, val string) {
+	envVar.Vars[key] = val
+}
+
+func (envVar EnvVar) get(key string) string {
+	return envVar.Vars[key]
+}
+
+func New(config ...Config) fiber.Handler {
+	var cfg Config
+	if len(config) > 0 {
+		cfg = config[0]
+	} else {
+		cfg = Config{Path: "/envvars"}
+	}
+
+	return func(c *fiber.Ctx) error {
+		path := c.Path()
+		if len(path) < len(cfg.Path) || !strings.HasPrefix(path, cfg.Path) {
+			return c.Next()
+		}
+
+		if path == cfg.Path {
+			envVar := newEnvVar(cfg)
+			varsByte, err := json.Marshal(envVar)
```

Comment:
Get this from app config instead of using hardcoded stdlib

Reply:
Thx, done 👍